contrib/install/local.cfg

```diff
@@ -0,0 +1,9 @@
+PACKAGES="load:dealii-prepare once:cmake once:astyle once:hdf5 once:p4est once:trilinos dealii"
+
+DEAL_II_VERSION=v9.4.2
```

Comment:
9.5 is already out, though perhaps not yet advertised: https://github.com/dealii/dealii/releases How about we use that right away? Then people won't have to update in a year.